core/arch/arm/plat-zynqmp/conf.mk

```diff
@@ -30,6 +30,10 @@ CFG_CRYPTO_WITH_CE ?= y
 
 CFG_ZYNQMP_PM ?= $(CFG_ARM64_core)
 
+ifeq ($(CFG_RPMB_FS),y)
+$(call force,CFG_ZYNQMP_HUK,y,Mandated by CFG_RPMB_FS)
```

Comment:
Isn't `CFG_ZYNQMP_HUK ?= $(CFG_RPMB_FS)` more correct? Or do you really want to force usage of HUK?

Reply:
yes, I wanted to force using the zynqmp HUK instead of some other value

Reply:
OK then